linkcheck/colorama.py

```diff
@@ -26,14 +26,13 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+from ctypes import (windll, byref, Structure, c_char, c_short, c_uint32,
+                    c_ushort, ArgumentError, WinError)
```

Comment:
Hey hey hey -- this is _not_ related to shit PR, but! -- hey! HEY!

I recently (re)learned about the existence of https://pypi.org/project/colorama/, and this looks like a vendored copy?

Can we pull it in via a dependency instead?  (Out of scope for this PR, obviously).

Reply:
```
# These functions are part of the python-colorama module
# They have been adjusted slightly for LinkChecker
```

Oh that is just lovely.  :(

Reply:
ugh another ticket?

Reply:
> ugh another ticket?

It is for Windows only:

ansicolor.py:62
```py
if os.name == 'nt':
    from . import colorama
```